algoliasearch/errors.go

```diff
@@ -10,6 +10,8 @@ var (
 	NoMoreSynonymsErr           error = errors.New("No more synonyms")
 	NoMoreRulesErr              error = errors.New("No more rules")
 	ExhaustionOfTryableHostsErr error = errors.New("All hosts have been contacted unsuccessfully")
+	SameAppIDErr                error = errors.New("Application IDs cannot be identical")
```

Comment:
IMO a better description: "The given indexes are on the same application. Use client.copy_index instead."

Reply:
Indeed, thank you for your careful feedback. Super good point 👍